go/billing/tasks.py

```diff
@@ -166,8 +174,10 @@ def get_channel_type(transaction, tagpools):
 def get_message_description(transaction):
     if transaction['message_direction'] == MessageCost.DIRECTION_INBOUND:
         return 'Messages received'
-    else:
+    elif transaction['message_direction'] == MessageCost.DIRECTION_OUTBOUND:
         return 'Messages sent'
+    else:
+        return None
```

Comment:
This isn't strictly necessary, but did lead to some confusion when debugging. Before this PR, we weren't differentiating between topup transactions and message transactions. Topup transactions don't have a message direction, so they were being treated as 'Messages sent' line item, since they followed the `else` path.

I can't think of any way this could possibly happen again, but thought it might save some confusion when trying to debug stuff if this did for some reason happen again (possibly because of some future code change).